packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -66,6 +72,7 @@ const FormLabel = ({
   id,
   htmlFor,
   size = 'medium',
+  componentName,
```

Comment:
Instead of componentName we can take `labelMarginBottom` type of prop so that we don't end up with too many component-specific checks here. Otherwise it becomes sort of a magic abstraction.

Reply:
I am guessing we need `labelMarginTop` instead 🤔